bcda/models/SwaggerStructs.go

```diff
@@ -106,9 +106,12 @@ type FileParam struct {
 // swagger:parameters bulkRequest
 type BulkRequestHeaders struct {
 	// required: true
-	// pattern: application/fhir\+json
+	// in: header
```

Comment:
Same with this file.  I wasn't expecting this file to change as part of BCDA-520.  Can you re-sync with `master`?